### DISCUSSION TITLE
Estimated Battery Range Strangeness

### DISCUSSION THREAD

**Discussion by drjjr2** [teslamate-org/teslamate#3162] [Q&A]
I've always used est_battery_range_km in my Home Assistant dashboards. Recently that number dropped dramatically. As if it is now being reported in miles and I'm still converting it as if it were kilometers. 2023 Tesla M3P on 2023.6.9. This might have started on 2023.6.8 on 03/23.

Here's what is reported

```
usable_battery_level = 90
ideal_battery_range_km = 448.19
est_battery_range_km = 245.36
rated_battery_range_km = 448.19
```

Any ideas?

**Comment by drjjr2** [comment]
Looks like it's a bug/feature in 2023.6.8 where they reworked the Energy App.
https://teslamotorsclub.com/tmc/threads/problems-with-%E2%80%9Cenergy-app%E2%80%9D.297543/#post-7511947

**Comment by ktsenvolt** [comment]
![image](https://user-images.githubusercontent.com/80360674/235328713-c863b0ad-c519-4616-bb34-ed8c14835d0d.png)
Issue on newer version also.
